### ISSUE TITLE
[Bug]: Raspberry Pi, interface issues (no icons), invisible elements, jumping layout

### ISSUE THREAD

**Issue by IvanGordeyev** [FreeTubeApp/FreeTube#4888]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [x] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [X] This issue contains only one bug.

### Describe the bug

1. Open application (interface already incorrect)

### Expected Behavior

Expected behaviour:
Interface is rendered correctly and showing icons and elements in the correct places.
![Screenshot from 2024-04-06 13-44-05](https://github.com/FreeTubeApp/FreeTube/assets/48790568/cf625616-17a5-4b02-8987-619ffa9fa275)

![Screenshot from 2024-04-06 13-43-00](https://github.com/FreeTubeApp/FreeTube/assets/48790568/b14b6623-ca5d-4511-b30d-6f4cf24c5949)

### Issue Labels

usability issue

### FreeTube Version

v0.20.0 Beta

### Operating System Version

Raspbian GNU/Linux 11 (bullseye) Like Debian

### Installation Method

other

### Primary API used

Local API

### Last Known Working FreeTube Version (If Any)

v0.19.0

### Additional Information

Elements are not visible but mostly located in the right places.
Rendering is buggy.
Installed using Pi-Apps
Updated 06/04/2024.
Please see the images.

At the time of this most

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by IvanGordeyev** [comment]
Just to confirm, I've now tried the official version, from freetubeapp.io and I am having the same issue.

**Comment by absidue** [comment]
Can you please confirm that it happens in an official installation method too, unless it happens with an official installation too, there is nothing we can do about it on our side. Please report the issue to the maintainers of your installation method instead if that is the case.

and here the part of the issue template that you skipped while reporting the issue

> When you select an unofficial installation method, you must have verified that the bug is also present in one of the official installation methods. Please make sure you uninstall the unofficial installation before installing one of the official installations. If you can't reproduce this in one of the official installation methods, you should report the bug to the maintainer of the unofficial installation method you used.

**Comment by absidue** [comment]
Ah looks like our comments were 36 seconds apart, you must have sent yours while I was typing, sorry about that.

Which Raspberry Pi model are you using? I know that v5 has no support for hardware accelerated video playback.

**Comment by IvanGordeyev** [comment]
Thank you for your reply.

Raspberry pi 4 Model B rev 1.2 - 4GB - 128mb for GPU (Raspian OS 32bit with default settings and few installed apps)
Raspberry pi 4 Model B rev 1.4 - 8GB - 128mb for GPU (Raspian OS 32bit with sound/bluetooth changes, but nothing GPU/video rendering related)

Both behaving similar.
Both have vc4-fkms-v3d enabled (to make HDMI output availble by default when plugged in, if needed).
I've tried to GPU memory allocation to 512 on one but nothing changes.

**Comment by MCFtm** [comment]
Hi there, I have the same issue. It came with the latest update.

**Comment by absidue** [comment]
@IvanGordeyev Have you checked in the task manager that FreeTube is actually using the GPU?

**Comment by IvanGordeyev** [comment]
@absidue 
Will check if I can shortly, also on fresh 64bit install.

**Comment by IvanGordeyev** [comment]
![GPU_64Bit_Screenshot from 2024-04-13 21-42-13](https://github.com/FreeTubeApp/FreeTube/assets/48790568/b080e441-060e-4165-8b9c-60fc105f5d34)
Interestingly, working on 64bit Raspberry pi OS, from official website.
Will try on fresh 32bit next.

**Comment by IvanGordeyev** [comment]
Working on 32bit fresh install.
Will try to upgrade current setup.

Latest version:
PRETTY_NAME="Raspbian GNU/Linux 12 (bookworm)"
NAME="Raspbian GNU/Linux"
VERSION_ID="12"
VERSION="12 (bookworm)"
VERSION_CODENAME=bookworm
ID=raspbian
ID_LIKE=debian
HOME_URL="http://www.raspbian.org/"
SUPPORT_URL="http://www.raspbian.org/RaspbianForums"
BUG_REPORT_URL="http://www.raspbian.org/RaspbianBugs"

My version:
PRETTY_NAME="Raspbian GNU/Linux 12 (bookworm)"
NAME="Raspbian GNU/Linux"
VERSION_ID="12"
VERSION="12 (bookworm)"
VERSION_CODENAME=bookworm
ID=raspbian
ID_LIKE=debian
HOME_URL="http://www.raspbian.org/"
SUPPORT_URL="http://www.raspbian.org/RaspbianForums"
BUG_REPORT_URL="http://www.raspbian.org/RaspbianBugs"

**Comment by IvanGordeyev** [comment]
Please consider closed. Working as expected after distribution version upgrade.
Did a fresh installed after reading about paths from bullseye to bookworm.

Please accept my apology for taking your time away.

Thank you for your time and thank you for the work on this great project.

Our use of Freetube is
- Background ambience/music for work
- Distraction free bedtime audiobooks for kids (essential use for us)

**Comment by MCFtm** [comment]
Didn't manage to bring freetube R20 up and running. Maybe the RPi 3A+ doesn't have enough power. Anyway, I reinstalled R19 which is running fine and good enough for my purposes. Thx

**Comment by archangel2222** [comment]
I have a rasoberry pi 4 8gb version running Bullseye 64bit OS and have tried all post v19 freetube versions and I have this issue not sure why. Looks exactly like the original screenshots provided everyone cut in to view. Was working fine until v19. How can I rollback and install v19 what command do I run I am not good at this. I can download the arm64 file but not sure how to install it yet. 

I have checked the OS updates and confirmed I updated freetube from pi-apps as well still no fix post v19
